pyproject.toml

```diff
@@ -13,6 +13,7 @@ grpcio = "<2"
 grpcio-tools = "<2"
 asyncio = "^3.4.3"
 proto-parser = "^1.6.3"
+protobuf = ">=3.5.0.post1,<4.0dev"
```

Comment:
why adding the constraint here ? grpcio-tools normally handle it

Reply:
It didn't work, I have some 4+ version and there are some changes in imports.

Reply:
Also I guess you should delete poetry.lock from the repo. I cannot build the docker with existing poetry.lock (Mac M1).

Reply:
Strange issue. Can you send me your current poetry.lock with a pastebin file ? So I can check the differences.
I do not have a Mac to reproduce but lock file should always be commited from my point of view.

Reply:
Unfortunately I don't have it already. Poetry.lock can be, but it shouldn't work on each platform in this case (not all builds exist for each platform).  So I need to remove it and build new one (compatible with Mac) via TOML file.

Reply:
Actually if you use docker image it should work regardless of the platform. Maybe adding the debian distribution to the docker file can help ?